run/image-processing/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+
+# Use the official maven/Java 11 image to create a build artifact.
+# https://hub.docker.com/_/maven
+FROM maven:3.6.1-jdk-11 as builder
```

Comment:
3.6.2 is out.

Reply:
You might consider explicitly using mirror.gcr.io as well.

Reply:
done.